pkg/probe/exec/exec.go

```diff
@@ -17,39 +17,102 @@ limitations under the License.
 package exec
 
 import (
+	"bytes"
+	"io"
+
+	"k8s.io/klog"
 	"k8s.io/kubernetes/pkg/probe"
 	"k8s.io/utils/exec"
+)
 
-	"k8s.io/klog"
+const (
+	maxReadBytes = 2 * 1024 // 2kb
```

Comment:
This limit should be documented in the exec probe API, and release notes.

Reply:
Definitely. This was just a random number I started experimenting with.